@hubblecommerce/hubble/core/components/detail/ViewProduct.vue

```diff
@@ -100,6 +100,13 @@ export default {
 
             return path;
         },
+        category: function () {
+            if (Object.keys(this.breadcrumb).length > 0) {
+                return this.breadcrumb[Object.keys(this.breadcrumb)[Object.keys(this.breadcrumb).length - 2]].name;
```

Comment:
Does this work with all kind of breadcrumb structure?

Reply:
Yes, it always returns the parent category.